docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -440,6 +440,8 @@ To add Semgrep into Azure Pipelines:
 
 ### Sample Azure Pipelines configuration snippet
 
+This configuration snippet is tested with hosted Azure runners. If you are using self-hosted runners, you may need to make adjustments to ensure that the necessary software is available.
```

Comment:
```suggestion
This configuration snippet is tested with hosted Azure runners. If you are using self-hosted runners, you may need to make adjustments to ensure that the necessary dependencies is available.
```

I don't have a strong preference, but maybe dependencies is more descriptive?

Reply:
I don't know that it's really more helpful here - I'm going to go with merging and we can revisit later if needed.